projects/ngx-scrollbar/smooth-scroll/src/smooth-scroll-manager.ts

```diff
@@ -143,14 +152,20 @@ export class SmoothScrollManager {
 
   private _applyScrollToOptions(el: HTMLElement, options: SmoothScrollToOptions): Promise<void> {
     if (!options.duration) {
-      this._scrollElement(el, options.left, options.top);
+      if(options.left !== undefined && options.top !== undefined) {
+        this._scrollElement(el, options.left!, options.top!);
+      }
       return Promise.resolve();
     }
 
     // Initialize a destroyer stream, reinitialize it if the element is already being scrolled
     const destroyed: Subject<void> = this._initSmoothScroll(el);
 
-    const easingOptions = options.easing || this._defaultOptions.easing;
+    let easingOptions: BezierEasingOptions =
+    options.easing? options.easing : this._defaultOptions.easing!; // assuming that easing can't be undefined
+    if (!easingOptions.x1 || !easingOptions.x2 || !easingOptions.y1 || !easingOptions.y2) {
+      easingOptions = this._defaultOptions.easing!; // assuming that easing can't be undefined
+    }
```

Comment:
Please test it without this change

Reply:
Again complaining because of potential undefined.

This compiles but is probably less safe (unless you're 100% sure it will never be null/undefined at this point:

`easing: BezierEasing(options.easing!.x1!, options.easing!.y1!, options.easing!.x2!, options.easing!.y2!)`